FWCore/MessageLogger/src/ELseverityLevel.cc

```diff
@@ -150,6 +110,7 @@ namespace edm {
       ret[ELsev_zeroSeverity] = "--";
       ret[ELsev_success] = "Debug";  // 4/21/09 mf
```

Comment:
Do you want to remove the comment? (I'm fine with either way)

Reply:
done